pkgs/desktops/plasma-5/kde-cli-tools.nix

```
@@ -11,4 +11,12 @@ mkDerivation {
     kcmutils kconfig kdesu ki18n kiconthemes kinit kio kwindowsystem qtsvg
     qtx11extras kactivities plasma-workspace
   ];
+  postInstall = ''
+    cat > $out/bin/kdesu <<EOF
+    #! /bin/sh -e
+    export PATH="\''${PATH}:${kinit}/bin"
+    exec -a "\$0" $out/libexec/kf5/kdesu "\$@"
+    EOF
+    chmod +x $out/bin/kdesu
```

The preferred way to do this is:

```suggestion
    makeWrapper $out/libexec/kf5/kdesu $out/bin/kdesu \
        --suffix PATH : ${lib.getBin kinit}/bin
```

This requires `makeWrapper` in `nativeBuildInputs`.